### PR TITLE
Send message upon sheet creation. Also rename task.py -> tasks.py

### PR DESCRIPTION
**Pull request by asdfryan**

### CODE REVIEW

File: google_api_lib/tasks.py [hunk 1]
```diff
@@ -50,8 +51,12 @@ def create_google_sheets(self, puzzle_id, name, puzzle_url=None):
     sheet_url = response["webViewLink"]
     if puzzle_url:
         add_puzzle_link_to_sheet(puzzle_url, sheet_url)
-    puzzle = Puzzle.objects.filter(pk=puzzle_id).update(sheet=sheet_url)
+    puzzle = Puzzle.objects.get(pk=puzzle_id)
+    puzzle.sheet = sheet_url
+    puzzle.save()
```
Comment (rgossiaux) [inline google_api_lib/tasks.py:56]: nit: i think the other way still works and is one fewer db query (update vs get + update, not 100% sure though)

**Review by rgossiaux — APPROVED** [review]
Nice, thanks!

File: google_api_lib/tasks.py (hunk 1 above)
Comment (asdfryan) [reply google_api_lib/tasks.py:56]: I needed the puzzle object to call `chat_room` to confirm existence.

Comment (rgossiaux) [reply google_api_lib/tasks.py:56]: Oops yup, didn't read closely, sounds good

**Pull request merged**